### ISSUE TITLE
awk error on ssh backend

### ISSUE THREAD

**Issue by hameno** [zertrin/duplicity-backup#103]
Hey,
I'm trying the latest master version on FreeBSD 9.3 with duplicity 0.6.25 and I'm getting the following error
```
awk: non-terminated string ") break; ... at source line 1
 context is
         >>>  <<<
awk: giving up
 source line number 2
```

**Comment by zertrin** [comment]
Hi, thanks for this bug report.

From what I can guess the problem is probably situated [line 446](https://github.com/zertrin/duplicity-backup/blob/master/duplicity-backup.sh#L446) of the script where the ssh options are parsed by awk.

I didn't contribute that part of the script and [a quick git blame](https://github.com/zertrin/duplicity-backup/blame/master/duplicity-backup.sh#L446) indicates that that line was changed for the last time in the commit c8d249da by @mddirba

I'll have another look at it asap, but as I'm not using the ssh backend myself, it may take some time. In the meantime @mddirba might have a look at it and maybe spot the problem more easily given he wrote that part ;)

**Comment by zertrin** [comment]
@hameno Additionnaly, could you provide:
  - what the variable `STATIC_OPTIONS` contains in your config file (redacting any potential personal information)
  - and which shell are you using (`bash`, `zsh`, `ash`, `ksh`...?) and which version of that shell do you use

**Comment by hameno** [comment]
Sure,
STATIC_OPTIONS were the default ones, so 
```STATIC_OPTIONS="--full-if-older-than 31D --s3-use-new-style"```
The shell is the default one for FreeBSD 9.3 (tcsh) and bash version is
```GNU bash, version 4.3.39(3)-release (amd64-portbld-freebsd9.3)```
awk version:
 ```awk version 20121220 (FreeBSD)```

**Comment by mddirba** [comment]
Would you mind pulling from me and trying out my fix?
https://github.com/mddirba/duplicity-backup/tree/awk_backslash
https://github.com/mddirba/duplicity-backup/commit/689eb2316186fa522073a7f0291a333ecb0a3b3b

**Comment by hameno** [comment]
I'll try it tomorrow

**Comment by hameno** [comment]
Yep, this seems to work.

**Comment by zertrin** [comment]
I've merged 689eb23 which should fix this bug.
Thanks @mddirba for the fix and @hameno for the feedback.
